gapis/api/vulkan/vulkan.go

```diff
@@ -358,6 +360,36 @@ func (API) ResolveSynchronization(ctx context.Context, d *sync.Data, c *path.Cap
 				}
 			}
 			d.SubcommandReferences[i] = refs
+			syncCouldHaveChanged = true
+		case *VkQueueBindSparse:
+			syncCouldHaveChanged = true
+		case *VkSignalSemaphoreKHR:
+			syncCouldHaveChanged = true
+		case *VkSignalSemaphore:
+			syncCouldHaveChanged = true
+		case *VkWaitSemaphoresKHR:
+			syncCouldHaveChanged = true
+		case *VkWaitSemaphores:
+			syncCouldHaveChanged = true
+		}
+		blocked := false
+		if syncCouldHaveChanged {
+			for _, q := range st.Queues().All() {
+				if q.PendingSubmissions().Len() > 0 {
+					blocked = true
+					break
+				}
+			}
+		}
+		if !blocked {
+			for j := 0; j < len(blockedCommands); j++ {
+				d.UnblockingCommands[blockedCommands[j]] = id
+			}
```

Comment:
should `blockedCommands` be cleared here?

Reply:
Yep, fixed.